index.html

```diff
@@ -19,8 +19,13 @@ <h1>Leave your footprint here!</h1>
 			<li><h3 class="ariel">Ariel was here!</h3></li>
 			<li><h3 class="ariel2">Ariel was here, again!</h3></li>
 			<li><h3 class="fred">Frederick was here!</h3></li>
+
+			<li><h3 class="serena">Serena was here!</h3></li>
```

Comment:
serena actually put this one in by accident. Remove this one and keep the one on line 26 (serena2)

Reply:
Alright, those should be fixed now.